cp23/hpi-fun2/motivation/executables/generator.py

```diff
@@ -0,0 +1,74 @@
+from pathlib import Path
+
+SAMPLES = [
+    [
+        "sample1",
+        """
+          11 20
+          2 5 3 7 2 1 10 4 1 8 2
+          1 2 2 2 5 5 1 8 8 8
+        """
+    ],
+    [
+        "sample2",
+        """
+          5 7
+          1 5 10 -2 -2
+          1 1 3 3
+        """
+    ],
+    [
+        "sample3",
+        """
+          5 7
+          8 5 10 -2 -2
+          1 1 3 3
+        """
```

Comment:
Please remove all leading and trailing spaces.

Reply:
Done 👍🏼